test/unit/layeredmaterialnodeprocessing.js

```diff
@@ -28,18 +28,23 @@ describe('updateLayeredMaterialNodeImagery', function () {
 
     const layer = {
         id: 'foo',
-        protocol: 'dummy',
-        extent: new Extent('EPSG:4326', 0, 0, 0, 0),
+        source: {
+            protocol: 'dummy',
+            extent: new Extent('EPSG:4326', 0, 0, 0, 0),
+        },
     };
 
     beforeEach('reset state', function () {
         // clear commands array
         context.scheduler.commands = [];
         // reset default layer state
-        layer.tileInsideLimit = () => true;
-        layer.visible = true;
+
         layer.updateStrategy = STRATEGY_MIN_NETWORK_TRAFFIC;
```

Comment:
Side note, but this should be `layer.updateStrategy = { type: STRATEGY_MIN_NETWORK_TRAFFIC }`.

Reply:
yes